src/mk_links.sh

```diff
@@ -40,15 +44,16 @@ recursive_setup() {
 								mv $home_twin $archi_path
 								break
 							else
-								printf "Error: another $base_file file is present in $archi_path. Aborting\n"
+								e_error "another $base_file file is present in $archi_path. Aborting\n"
 								exit 1
 							fi
 							;;
 						y)
 							break
 							;;
 						*)
-							printf 'Invalid choice. Try again, shall I overwrite it ? (y=yes n=no a=archive) '
+							e_error "Invalid choice."
+							e_prompt "Try again, shall I overwrite it ? (y=yes n=no a=archive)"
```

Comment:
I have not tried it but I think it will appear as "Try again, shall I overwrite it **?** (y=yes n=no a=archive)**?**", that is, with one too many exclamation marks.

Oh by the way, the mistake was there before your PR but could you remove the space between `it` and the exclamation mark?